src/write_outputs/write_costs.jl

```diff
@@ -109,7 +196,11 @@ function write_costs(path::AbstractString, inputs::Dict, setup::Dict, EP::Model)
 			tempCNSE *= ModelScalingFactor^2
 			tempCStart *= ModelScalingFactor^2
 		end
-		dfCost[!,Symbol("Zone$z")] = [tempCTotal, tempCFix, tempCVar, tempCNSE, tempCStart, "-", "-", "-"]
+		if !isempty(VRE_STOR)
+			dfCost[!,Symbol("Zone$z")] = [tempCTotal, tempCFix, tempCVar, tempCNSE, tempCStart, "-", "-", "-", "-"]
+		else
+			dfCost[!,Symbol("Zone$z")] = [tempCTotal, tempCFix, tempCVar, tempCNSE, tempCStart, "-", "-", "-"]
+		end
```

Comment:
Similar to another comment above.
1) Initialize base vector of values
2) if !isempty(VRE_STOR) push an element to that vector
3) set that line or column or whatnot of dfCost to the vector